src/ir_NEC.h

```diff
@@ -14,6 +14,8 @@
 //   Brand: Yamaha,   Model: RXV585B A/V Receiver
 //   Brand: Aloka,    Model: SleepyLights LED Lamp
 //   Brand: Toshiba,  Model: 42TL838 LCD TV
+//   Brand: Duux,  Model: Blizzard Smart 10K / DXMA04 A/C
+//   Brand: Duux,  Model: YJ-A081 TR Remote
```

Comment:
Also maybe add reference to sc6121 IC as per https://github.com/crankyoldgit/IRremoteESP8266/issues/1176#issuecomment-640186591

Reply:
Done.

> Was thinking the same thing...
> Maybe a reference in Trotec to the issue, since it "happens to work" rather than really supported?

Reference added. It does control the device, just not "perfectly" which can be said for almost every A/C we support.